internal/console/helpers/helpers.go

```diff
@@ -213,3 +249,39 @@ func FileAnalyzer(path string) (string, error) {
 
 	return "", errors.New("invalid configuration file format")
 }
+
+func sortBySev(queries []model.VulnerableQuery) []model.VulnerableQuery {
```

Comment:
I think this fits better on model package, because it is relative to model, not to console package and we could use in others package as a helper function...

Reply:
resolved